spec/lexers/syzlang_spec.rb

```diff
@@ -0,0 +1,554 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+describe Rouge::Lexers::Syzlang do
+  let(:subject) { Rouge::Lexers::Syzlang.new }
+
+  describe 'lexing' do
+    include Support::Lexing
+
+    it 'recognizes comments' do
+      assert_tokens_equal '# comment',
+        ['Comment', '# comment']
+    end
+
+    it 'recognizes includes' do
+      assert_tokens_equal 'include <uapi/linux/a.out.h>',
+        ['Keyword', 'include'],
+        ['Text', ' '],
+        ['Punctuation', '<'],
+        ['Literal.String', 'uapi/linux/a.out.h'],
+        ['Punctuation', '>']
+    end
+
+    it 'recognizes defines' do
+      assert_tokens_equal 'define FOO (~BAR + 0x10) / 2',
```

Comment:
**suggestion:** We tend not to write specs for lexer this way since asserting specific tokens make the tests very brittle. Could we add these samples in `spec/visual/samples/syzlang` and verify via the visual tester (rackup and visit `http://localhost:9292`)